importexport/musicxml/exportxml.cpp

```diff
@@ -4347,6 +4347,9 @@ void ExportMusicXml::pedal(Pedal const* const pd, int staff, const Fraction& tic
                               pedalType = "resume";
                               break;
                               }
+                        else {
```

Comment:
The else is not needed. Not checked whether it harms though

Reply:
It shouldn't harm; I added it for clarity